source/standards/production-readiness.html.md.erb

```diff
@@ -16,11 +16,11 @@ Read the [Service Manual][] for more information about building and running a li
 
 ## Use a checklist
 
-To avoid problems when your service goes into production use a checklist to confirm if your service is ready. Each check should reference a standard that’s already in place before you can put a service into production. An example is, [documenting architecture decisions].
+To avoid problems when your service goes into production, use a checklist. Each check should reference a standard that’s already in place before you can put a service into production. An example is [documenting architecture decisions].
```

Comment:
Not entirely sure what this means: 'Each check should reference a standard that’s already in place before you can put a service into production.' Can it be reworded?